.devcontainer/devcontainer.json

```diff
@@ -27,7 +30,7 @@
 	},
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "sudo dpkg-reconfigure tzdata -f none && sudo mkdir -p /workspaces/Nuitka-website/.devcontainer/supervisor/logs && sudo chown -R vscode:vscode /workspaces/Nuitka-website/.devcontainer/supervisor",
+	"postCreateCommand": "sudo dpkg-reconfigure tzdata -f none && sudo mkdir -p /workspaces/Nuitka-website/.devcontainer/supervisor/logs && sudo chown -R vscode:vscode /workspaces/Nuitka-website/.devcontainer/supervisor && npm install",
```

Comment:
This one will conflict, as I changed this over to a list and added stuff recently here, but I can handle that